packages/shared/src/components/sidebar/MobileMenuIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useContext } from 'react';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import SettingsContext from '../../contexts/SettingsContext';
+import { Button } from '../buttons/Button';
+import ArrowIcon from '../icons/Arrow';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+
+export function MobileMenuIcon(): ReactElement {
+  const { trackEvent } = useContext(AnalyticsContext);
+  const { sidebarExpanded, toggleSidebarExpanded } =
+    useContext(SettingsContext);
```

Comment:
We can actually also pass these down form the sidebar as they already loaded there.